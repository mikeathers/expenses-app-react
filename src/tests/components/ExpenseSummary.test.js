import React from "react";
import { shallow } from "enzyme";

import ExpenseSummary from "../../components/ExpenseSummary";

test("should render ExpenseSummary correctly", () => {
  const wrapper = shallow(<ExpenseSummary />);
  expect(wrapper).toMatchSnapshot();
});