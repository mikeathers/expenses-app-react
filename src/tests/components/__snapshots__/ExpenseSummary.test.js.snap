// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseSummary correctly 1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h1
      className="page-header__title"
    >
      Expense Forms
    </h1>
    <button
      className="button"
      onClick={[Function]}
    >
      Add Expense Form
    </button>
    <ToggleDisplay
      show={false}
      tag="span"
    >
      <Connect(AddExpenseForm) />
    </ToggleDisplay>
  </div>
</div>
`;
